docker/Dockerfile.sidecar.ubuntu

```diff
@@ -32,6 +32,8 @@ RUN set -ex \
     \
     && rm -rf /tmp/filebeat*
 
+RUN apt-get update; apt-get upgrade -y
+RUN apt-get install -y net-tools
```

Comment:
You don't need to install anything. IP address can be obtained in a simpler way, as shown below.